PowerUp/app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,8 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name="powerup.systers.com.AboutActivity"
+            android:label="Power Up"></activity>
```

Comment:
Don't use harcoded strings for labels. Instead put the string on `strings.xml` and reference it here using `@string/string_name`